strings/tips.txt

```diff
@@ -57,9 +55,8 @@ As the Chief Engineer, you can spy on and even forge PDA communications with the
 As an Engineer, the supermatter shard is an extremely dangerous piece of equipment: touching it will disintegrate you.
 As an Engineer, you can electrify grilles by placing wire "nodes" beneath them: the big seemingly unconnected bulges from a half completed wiring job.
 As an Engineer, return to Engineering once in a while to check on the singularity or energy ball and the SMES cells. It's always a good idea to make sure containment isn't compromised.
-As an Engineer, you can power the station solely with the solar arrays. While uninteresting, it is a much safer alternative to the singularity engine.
+As an Engineer, you can power the whole ship using P.A.C.M.A.N. generators. Great for when the engine explodes!
 As an Engineer, you can repair windows by using a welding tool on them while on any intent other than harm.
-As an Engineer, your construction permit allows you to create a new area, but has only one use.
```

Comment:
Readded it but I accidentally placed it a few lines down so this change thing wont pick it up. Woops